### ISSUE TITLE
Dependency to vulnerable version of debug

### ISSUE THREAD

**Issue by smichaelsen** [localtunnel/localtunnel#251]
localtunnel depends on version `2.6.8` of the `debug` package, which has know vulnerability since last September 2017: https://nodesecurity.io/advisories/534

The dependency should be raised to at least 2.6.9

**Comment by defunctzombie** [comment]
I don't think we pass untrusted user input into the formatter so this is a non-issue. I'll look to upgrade it next cycle of package upgrades. Thanks for the notice.
